RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
+2.13
+-----
+- Pressing Shift + Tab will now de-indent items in a list. Thanks @sergiou87! #946
+
 2.12
 -----
-- Pressing Shift + Tab will now de-indent items in a list #864
+- Fixed a bug that prevented the app window from being moved #940
```

Comment:
This diff is a bit confusing. Here's what happened:

- There were no notes for 2.12, so I added one
- In the meantime, on `develop`, #946 went in, and the author, an external contributor 💙 , updated the file on the what looked like the appropriate spot at the time: the 2.12 slot.